tests/bdd/features/helm_chart/install.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Install Mayastor
+  Install Mayastor on a kubernetes cluster, using helm
+
+  Background:
+    Given: A Kubernetes cluster
+
+  Scenario: Mayastor helm chart install
+    Given worker nodes on the kubernetes cluster are labelled with the label 'openebs.io/engine=mayastor'
```

Comment:
```suggestion
    Given the worker nodes are labelled with the label 'openebs.io/engine=mayastor'
```

Reply:
Resolving this conversation, because this test is no longer a part of this PR.